clang/test/Driver/flang/dev_null_as_input_windows.f95

```diff
@@ -0,0 +1,11 @@
+! REQUIRES: classic_flang
+! REQUIRES: system-windows
+
+! Check that the driver invokes flang1 correctly for preprocessed fixed-form
+! Fortran code.
+
+! RUN: %clang --driver-mode=flang -E -target x86_64-unknown-windows-gnu -c - </dev/null -### 2>&1 \
```

Comment:
you can't use /dev/null for windows test...

Reply:
@pawosm-arm I just changed it from `-</dev/null` to `-<%s`
Since I am working on linux i cannot double check its working but in my opinion it should because it should have the same results.
Please take a look. Thanks

Reply:
@bryanpkc Just changed it to Your recommendation. Thanks